### PR TITLE
[RainDropFansub] add website support (#1624)

### LINKED ISSUE #1624: add raindropfansub
> **From:** 

**Version:** [](https://github.com/manga-download/hakuneko/commits/)

http://okuma.raindropfansub.com/
manga site in Turkish
theme: my manga reader

<details><summary>What has been done?</summary>

</details>

### PR DESCRIPTION
**Pull request by dodieboy**
Resolves #1624

### CODE REVIEW

File: src/web/mjs/connectors/RainDropFansub.mjs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import MangaReaderCMS from './templates/MangaReaderCMS.mjs';
+
+export default class RawMangaUpdate extends MangaReaderCMS {
```
Comment (27136773) [inline src/web/mjs/connectors/RainDropFansub.mjs]: The classname does not match the website snd should be changed

**Review by 27136773 — COMMENTED** [review]
Looks good, but classname should be adjusted before merge

File: src/web/mjs/connectors/RainDropFansub.mjs (hunk 1 above)
Comment (dodieboy) [reply src/web/mjs/connectors/RainDropFansub.mjs]: sry. I will make the change

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**